analytics/article_impressions.js

```diff
@@ -38,10 +38,34 @@ if(location.pathname.match('/article/') || location.pathname.match('/articles'))
               id: 'magazine_fixed'
             }
           }
+        }else if(classList.contains('article-es-form')){
+          var article = $(this).parent().parent().parent().parent();
```

Comment:
This isn't a very reliable selector! What would happen if we changed the order of parents or even more likely, if we decide to remove some container we don't need? We'll want to find use `parents` or `closest` instead. See https://www.sitepoint.com/jquerys-closest-parents/